etc/visual-studio/ot-ncp-uart.vcxproj

```diff
@@ -43,6 +43,7 @@
         OPENTHREAD_MULTIPLE_INSTANCE;
         OPENTHREAD_ENABLE_NCP_SPI=0;
         OOPENTHREAD_ENABLE_NCP_UART=1;
+        OPENTHREAD_ENABLE_BORDER_AGENT_PROXY=0;
```

Comment:
Could we move this to the config file instead of defining it in each project file?